index.yaml

```diff
@@ -0,0 +1,23 @@
+indexes:
+
+# AUTOGENERATED
+
+# This index.yaml is automatically updated whenever the dev_appserver
+# detects that a new type of query is run.  If you want to manage the
+# index.yaml file manually, remove the above marker line (the line
+# saying "# AUTOGENERATED").  If you want to manage some indexes
+# manually, move them above the marker line.  The index.yaml file is
+# automatically uploaded to the admin console when you next deploy
+# your application using appcfg.py.
+
+- kind: Searches
+  properties:
+  - name: private
+  - name: timestamp
+    direction: desc
+
+- kind: Searches
+  properties:
+  - name: user_id
+  - name: timestamp
+    direction: desc
```

Comment:
What is the purpose of these? I highly doubt that it is supported in Python3 runtime.

Reply:
It was automatically generated while running dev_appserver for Python2 local testing. I think it should be removed.